Makefile

```diff
@@ -63,14 +63,17 @@ ifndef PYLINT_CONCURRENCY
 	PYLINT_CONCURRENCY := 1
 endif
 
-NOSE_OPTS := --rednose --immediate --with-parallel --nocapture
+# NOTE: We exclude resourceregistrar DEBUG level log messages since those are very noisy (we
+# loaded resources for every tests) which makes tests hard to troubleshoot on failure due to
+# pages and pages and pages of noise.
+NOSE_OPTS := --rednose --immediate --with-parallel --nocapture --logging-filter=-st2.st2common.bootstrap
```

Comment:
The minus sign in front of `st2.st2common.bootstrap` is unexpected.

Can you add a link to the nose documentation, and/or a quote from that explaining what it is and what it is doing?

```suggestion
# The minus in front of st2.st2common.bootstrap filters out logging statements from that module.
# See https://nose.readthedocs.io/en/latest/usage.html#cmdoption-logging-filter
NOSE_OPTS := --rednose --immediate --with-parallel --nocapture --logging-filter=-st2.st2common.bootstrap
```

Reply:
This output has been bugging me for a long time (it just causes tons of noise for every test and makes easier issues harder to spot).

But yeah, it's nose specific syntax, will add a comment.